contracts/access/mod.go

```diff
@@ -0,0 +1,172 @@
+// Package access implements a native contract to handle access. It allows an
+// authorized identity to add access as an {ID, CONTRACT, COMMAND, IDENTITIES}
+// quadruplet.
+//
+// ID is the credential identifier. This identifier is generally defined at the
+// contract's creation.
+// CONTRACT is the contract name.
+// COMMAND specifies the command to grant access to on the contract.
+// IDENTITIES is a list of standard base64 encoded bls public keys, separated by
+// comas.
+package access
+
+import (
+	"encoding/base64"
+	"encoding/hex"
+	"strings"
+
+	"go.dedis.ch/dela"
+	"go.dedis.ch/dela/core/access"
+	"go.dedis.ch/dela/core/execution"
+	"go.dedis.ch/dela/core/execution/native"
+	"go.dedis.ch/dela/core/store"
+	"go.dedis.ch/dela/crypto/bls"
+	"golang.org/x/xerrors"
+)
+
+const (
+	// ContractName is the name of the access contract.
+	ContractName = "go.dedis.ch/dela.Access"
+
+	// GrantIDArg is the argument's name in the transaction that contains the
+	// provided id to grant
+	GrantIDArg = "access:grant_id"
+
+	// GrantContractArg is the argument's name in the transaction that contain
+	// the provided contract name to grant the access to.
+	GrantContractArg = "access:grant_contract"
+
+	// GrantCommandArg is the argument's name in the transaction that contains
+	// the provided command to grant access to.
+	GrantCommandArg = "access:grant_command"
+
+	// IdentityArg is the argument's name in the transaction that contains the
+	// provided identity to grant access to.
+	IdentityArg = "access:identity"
+
+	// CmdArg is the argument's name to indicate the kind of command we want to
+	// run on the contract. Should be one of the Command type.
+	CmdArg = "access:command"
+)
+
+// Command defines a command for the command contract
+type Command string
+
+const (
+	// CmdSet defines the command to grant access
+	CmdSet Command = "GRANT"
+)
+
+// NewCreds creates new credentials for an access contract execution.
+func NewCreds(id []byte) access.Credential {
+	return access.NewContractCreds(id, ContractName, "all")
```

Comment:
same remark as the other PR, especially given @gnarula had the same question.
It might be good to put that "all" in a named constant for clarity.